animation_url/src/routes/[slug]/+page.ts

```diff
@@ -90,58 +71,13 @@ const getPilot = async function (rigId: string): Promise<Pilot | undefined> {
       session.pilot_id,
       NftTokenType.ERC721
     );
-    console.info("pilot token:", pilotToken);
-
-    if (pilotToken.media.length > 0) {
-      const media = pilotToken.media[0];
-      const pilot: Pilot = {
-        uri: encodeSVG(media.thumbnail || media.gateway || media.raw),
-      };
-      if (media.format) {
-        pilot.type = mediaTypes.get(media.format);
-      } else {
-        pilot.type = getMediaType(pilot.uri);
-      }
-      return pilot.type ? pilot : { uri: unknown, type: MediaType.image };
-    } else if (pilotToken.rawMetadata?.image_data) {
-      return {
-        uri: pilotToken.rawMetadata?.image_data,
-        type: MediaType.image,
-      };
-    } else if (pilotToken.rawMetadata?.svg_image_data) {
-      return {
-        uri: pilotToken.rawMetadata?.svg_image_data,
-        type: MediaType.image,
-      };
-    } else {
-      return { uri: unknown, type: MediaType.image };
-    }
-  }
 
-  // else show trainer
-  return { uri: trainer, type: MediaType.image };
-};
+    const imageUrl = pilotToken?.media[0]?.gateway;
+    const rawSvg = pilotToken?.rawMetadata?.svg_image_data;
```

Comment:
happy to get rig of all the ugliness here, but...

- i have seen cases where `gateway` is empty, but `raw` has a value
- worth using thumbnail? some image are many mbs
- `image_data` can be present instead of `svg_image_data` according to OS metadata standards 

why not,

```
const imageUrl = pilotToken?.media[0]?.thumbnail || pilotToken?.media[0]?.gateway || pilotToken?.media[0]?.raw;
const rawSvg = pilotToken?.rawMetadata?.svg_image_data || pilotToken?.rawMetadata?.image_data;
```

`encodeSVG` was used to handle giant SVGs (like crypto punks). for some reason, they do not render unless they are base64 encoded... seems to be related to being giant.

Reply:
Ah I have never seen the case where gateway is empty and raw has a value. We should totally figure out what gives us the most compatibility, I just threw this together quick this morning because there are currently a few pilots that are broken in prod like moonbirds (because they use `https://gate.way/id` urls without a file ending), and since this is what we currently have in the garage I just know that that works for moonbirds at least

Reply:
sounds good. just sat down for a few min here. let's just merge this and I can follow up with a little tweak